cob_hardware_config/cob3-2/config/robot_modules.yaml

```diff
@@ -0,0 +1 @@
+robot_modules: ['base','tray','torso','head','arm','sdh']
```

Comment:
I am not sure now, but I think that we wanted to use it for teleop. I have to check it